source/documentation/other-topics/migrate-to-live-sa-github-actions.html.md.erb

```diff
@@ -0,0 +1,65 @@
+---
+title: Migrating serviceaccount module for GitHub actions to the "Live" cluster
+last_reviewed_on: 2021-09-17
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+## Migrating service account module
+
+
+#### Skip file to your namespace in live-1
+
+Migrating your environment using [cloud-platform-cli][migrate-cli] will add a [skip your namespace][long-running] file in live-1, This will allow the Apply Pipeline to skip
+live-1 namespace during migration, make sure you have this skip file available in your "live-1" namespace.
+
+#### Continuous Deployment of an application using Github actions in live
```

Comment:
Can we say “Live” and "Live-1" as in within quotes wherever we refer the cluster